setup.py

```diff
@@ -3,14 +3,13 @@
 """Setup configuration."""
 
 from setuptools import setup, find_packages
-import ephypype
 
-VERSION = ephypype.__version__
 
 if __name__ == "__main__":
     setup(
         name='ephypype',
-        version=VERSION,
+        # version=VERSION,
+        version='0.1.dev0',
```

Comment:
why? I had fixed this ...

Reply:
so the correct way to do this would be: https://github.com/mne-tools/mne-python/blob/master/setup.py#L13-L20

You need `version` in your `__init__` so that when you do `import ephypype` you can do `ephypype.__version__`

Reply:
ok, I see

Reply:
Should I create another PR?

Reply:
yep, go ahead!

Reply:
wait, I may have been to quick earlier and did not wait  the full  test, and now it is still building 
https://github.com/neuropycon/ephypype/pull/24

Reply:
Is it dead or it always takes so long? I mean the build.